debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -486,6 +499,29 @@ protected Object convertHStore(Column column, Field fieldDefn, Object data, HSto
             return convertHstoreToMap(column, fieldDefn, data);
         }
 
+    private Object convertLtree(Column column, Field fieldDefn, Object data) {
+        return convertValue(column, fieldDefn, data, "", r -> {
+            if (data instanceof String) {
+                r.deliver(data);
```

Comment:
For now, the data is delivered as `java.lang.String`. Should we consider using a real Java representation, such as `Ltree`? This class can contain functions shown in PostgreSQL, see [PostgreSQL Documentation - F.14. ltree Functions](https://www.postgresql.org/docs/current/ltree.html#LTREE-FUNC-TABLE). What I imagine is:

```java
r.deliver(Ltree.parse(data));
```

or

```
r.deliver(new Ltree(data));
```

where "data" is the label of the ltree.

Reply:
I don't think that's needed. In the end, the value would be propagated as a string anyways.